src/qml/models/nodemodel.cpp

```diff
@@ -166,3 +166,13 @@ void NodeModel::ConnectToNumConnectionsChangedSignal()
             setNumOutboundPeers(new_num_peers.outbound_full_relay + new_num_peers.block_relay);
         });
 }
+
+bool NodeModel::validateProxyAddress(QString ipAddress)
```

Comment:
Variable names should use snake_case, so `ip_address`, but maybe `addr_port` or even just `proxy` or `proxy_str` is a better name, especially if `unix:/path/to/socket` is to be accepted.

Reply:
True, ok.

Reply:
>  especially if unix:/path/to/socket is to be accepted.

As explained [above](https://github.com/bitcoin-core/gui-qml/pull/430#discussion_r1829543225), unix sockets support will be addressed in a follow-up once the feature is ready in this repo.